.github/workflows/release_staging.yml

```diff
@@ -0,0 +1,76 @@
+name: Deriv App
+on:
+  push:
+    branches:
+    - master
+env:
+  CACHE_VERSION: ${{ secrets.CACHE_VERSION }}
```

Comment:
Need to add all of the secrets to github secrets in order to make it work.
We may not need some of them which I remove them one by one,
Any hint would be appreciated